docs/community/README.md

```diff
@@ -25,11 +25,9 @@ Casual chat fan? Here are [all our channels](chat.md). There's getting-started a
 
 ## Community Calendar
 
-If you're an IPFS implementer, consumer, or user, you are welcome to join the community meetings we host regularly.
+If you're an IPFS implementer, application developer, or user, you are welcome to join the community meetings we host regularly.
 
-Stay up-to-date by subscribing to the [**IPFS Community Calendar**](https://calendar.google.com/calendar/embed?src=ipfs.io_eal36ugu5e75s207gfjcu0ae84%40group.calendar.google.com) and joining community meetings such as triage sessions, the IPFS implementers sync, and the IPLD sync.
```

Comment:
_Suggested copy edit:_

If you're an IPFS implementer, application developer, or user, you are welcome to join the community meetings we regularly host. Meetings include triage sessions, the IPFS implementers sync, and the IPLD sync.

For an up-to-date information, subscribe to the [**IPFS Community Calendar**](https://calendar.google.com/calendar/embed?src=ipfs.io_eal36ugu5e75s207gfjcu0ae84%40group.calendar.google.com).